manifests/params.pp

```diff
@@ -29,9 +29,18 @@
   if $::operatingsystem == 'Ubuntu' {
     if versioncmp($::lsbdistrelease, '8.04') < 1 {
       $init_style = 'debian'
-    } else {
+    } elsif versioncmp($::operatingsystemrelease, '15.04') < 0 {
       $init_style = 'upstart'
     }
+    else {
```

Comment:
That's true, you're probably both right.  I've often thought of puppet as trying to cover all the bases by default to reduce the amount of conditionals that have to exist outside of the puppet module itself when it comes to OS compatibility, but at the same time, this may be one abstraction too far as it's likely a fairly edge case where I was trying to support the outlying resistant to systemd crowd while it is only a few and this module already exposes the option to override $init_style, so it isn't surgery to make it work from the outside for the stragglers.  I'll clear up the is_systemd stuff.  I'm also going to bump the lsbdistrelease over to operatingsystemrelease for Ubuntu 8's line as that value has been around long enough in facter, it was just operatingsystemmajrelease that wasn't it seems.